app/operations/slack/send_daily_messages.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Slack
+  class SendDailyMessages
+    def call
+      companies_events.each do |companie_events|
```

Comment:
`companies_events.each do |companie_events|`
replace to
`companies_events.each do |company, events|`